src/index.html

```diff
@@ -1,13 +1,401 @@
+                <img
+                  src="./images/people/scarlett.png"
+                  alt="Leslie Mckinney"
+                  class="testimonial__photo"
+                >
+              </div>
+              <h3 class="testimonial__name">
+                <a href="#" class="link">
+                  Scarlett Hawkins
+                </a>
+              </h3>
+              <div class="testimonial__role">
+                Senior Vice President
+              </div>
+              <div class="testimonial__about">
+                My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart
+              </div>
+            </div>
+
+            <div class="
+                  testimonial
+                  testimonials__item
+                  grid__item
+                  grid__item--9-12"
+            >
+              <div class="testimonial__photo-container">
+                <img
+                  src="./images/people/max.png"
+                  alt="Leslie Mckinney"
+                  class="testimonial__photo"
+                >
+              </div>
+              <h3 class="testimonial__name">
+                <a href="#" class="link">
+                  Max Henry
+                </a>
+              </h3>
+              <div class="testimonial__role">
+                Chief Design Officer
+              </div>
+              <div class="testimonial__about">
+                When my lovely valley of steam rises and half-day sun is above an impermeable
+              </div>
+            </div>
+
+
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__label">
+                  Phone
+                </div>
+                <div class="contact-us__link">
+                  <a href="tel: +1 234 5555-55-55" class="link link--input">+1 234 5555-55-55</a>
+                </div>
+              </div>
+              <div class="contact-us__info">
+                <div class="contact-us__label">
+                  Email
+                </div>
+                <div class="contact-us__link">
+                  <a href="mailto: hello@miami.com" class="link link--input">
+                    hello@miami.com
+                  </a>
+                </div>
+              </div>
+            </div>
+            <div class="grid__item grid__item--4-5">
+              <div class="contact-us__info">
+                <div class="contact-us__label">
+                  Address
+                </div>
+                <div class="contact-us__link">
+                  <a href="mailto: hello@miami.com" class="link link--input">
```

Comment:
This link should lead to the location on google maps. Change href value to correct one. Also don't forget to make it open in the new tab.